index.js

```diff
@@ -25,12 +25,13 @@ class MarkdownButtonElement extends HTMLElement {
 
   connectedCallback() {
     if (!this.hasAttribute('tabindex')) {
-      this.setAttribute('tabindex', '0')
+      this.setAttribute('tabindex', '-1')
     }
 
     if (!this.hasAttribute('role')) {
       this.setAttribute('role', 'button')
     }
+    this.setAttribute('data-md-button', '')
```

Comment:
Turns out this isn't great for these default buttons, since we can't guarantee they get upgraded before markdown-toolbar. If they get upgraded after markdown-toolbar then there wouldn't be a first `data-md-button`. Perhaps we'll need to keep a list of button selectors?